components/library/libadwaita/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Geoff Weiss
+#
+
+include ../../../make-rules/shared-macros.mk
+ENV= /usr/bin/env
+
+COMPONENT_NAME=		libadwaita
+COMPONENT_VERSION=	1.5.4
+COMPONENT_SUMMARY=	Building blocks for modern GNOME applications.
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	sha256:1c7ebf9d03fbd50485831099c6a04670dc5399a5a832d900832301dbf22f9589
+COMPONENT_ARCHIVE_URL=	https://download.gnome.org/sources/$(COMPONENT_NAME)/1.5/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	https://gnome.pages.gitlab.gnome.org/libadwaita/doc/
+COMPONENT_LICENSE=	LGPLv2.1+
+COMPONENT_LICENSE_FILE=	COPYING
+COMPONENT_CLASSIFICATION=	Desktop (GNOME)/Libraries
+COMPONENT_FMRI=		library/desktop/libadwaita
+
+BUILD_STYLE=		meson
+
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
Include of `gnome.mk` above this line would allow you to remove several `COMPONENT_*` lines above.  For example `COMPONENT_SRC`, or `COMPONENT_ARCHIVE`.